scripts/fix-github-md-refs.sh

```diff
@@ -1,28 +1,47 @@
 #!/bin/bash
+# Copyright (C) 2024 Intel Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
 # We'll post process the markdown files copied to the _build/rst folder to look
 # for hard references to the github.com markdown file, for example
 # (https://github.com/opea-project/.../blob/.../README.md) and make them
 # relative to the _build/rst directory structure where docs are being built
 
 # Work on the current directory or the directory passed as the first argument
-# (as done in the makefile)
+# (as done in the makefile).  Normally is _build/rst
 
 cd ${1:-.}
 
-files=`grep -ril --include="*.md" 'github.com/opea-project.*\/[^\)]*\.md'`
+# look for markdown files containing a hard github.com/opea-project/...
+# reference to a markdown file
+
+mdfiles=`grep -ril --include="*.md" 'github.com/opea-project.*\/[^\)]*\.md'`
```

Comment:
are we missing a "(" in the regular expression?

Reply:
Nope.  That pattern is looking for a string of characters that's not an open parenthesis.